src/app/globals.css

```diff
@@ -70,3 +70,22 @@ body {
     @apply bg-background text-foreground;
   }
 }
+
+h1,
+h2,
+h3 {
+  font-family: "Roboto", sans-serif;
+  line-height: 1.2;
+}
+
+h1 {
+  font-size: 2.25rem;
+}
+
+h2 {
+  font-size: 1.875rem;
+}
+
+h3 {
+  font-size: 1.5rem;
+}
```

Comment:
hardcoding styles for tags might be a problem later. note for future - add our custom classes for text sizes in tailwind theme config